README.rst

```diff
@@ -19,10 +19,12 @@ Quick examples
 Quick start
 ::
 
-  from pyope.ope import OPE
-  cipher = OPE(b'key goes here')
+  from pyope.ope import OPE, generate_key
+  k = generate_key() # generate random key
+  cipher = OPE(k)
   assert cipher.encrypt(1000) < cipher.encrypt(2000) < cipher.encrypt(3000)
   assert cipher.decrypt(cipher.encrypt(1337)) == 1337
+  assert cipher.encrypt(1337) + cipher.encrypt(1) < cipher.encrypt(2000) # basic addition/subtraction works too!
```

Comment:
Addition and subtraction work because outputs are integers, but generally speaking, `x + y < z` does not imply that `encrypt(x) + encrypt(y) < encrypt(z)`. Take an example:
```
>>> from pyope.ope import OPE, ValueRange
>>> cipher = OPE('123', in_range=ValueRange(0,100), out_range=ValueRange(100,200))
>>> cipher.encrypt(1)
101
>>> cipher.encrypt(2)
102
>>> cipher.encrypt(4)
104
>>> cipher.encrypt(1) + cipher.encrypt(2) < cipher.encrypt(4)
False
```

Therefore, I don't see much sense in adding this change, it might confuse users.